portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,4 +67,11 @@ private String convertToJsonUsingGson() {
     String json = gson.toJson(messages);
     return json;
   }
+
+  private Comment getComment(HttpServletRequest request) {
```

Comment:
getComment sounds like it is supposed to retrieve the comment from the datastore. Can this be called something like makeComment or createComment or parseRequestIntoComment or something along those lines.